bHY6yh-de5998/cod-yyyiou_bHY6yh-de5998.xml

```diff
@@ -0,0 +1,129 @@
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-14" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="9-r"/>
+            <cb ed="#C" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de5998"><!-- p1l1-r21 -->
+            <head xml:id="bHY6yh-de5998-Hd1e108">Rubrica 21</head>
+            <head xml:id="bHY6yh-de5998-Hd1e111" type="question-title">De decimarum et primitiarum religione</head>
+            <p xml:id="bHY6yh-de5998-d1e114">
+               [omitted]
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e121">
+              IN exodo 22<quote xml:id="bHY6yh-de5998-Qd1e125"> Decimas tuas et primitias
+              <lb ed="#C"/>non tardabis auferre</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e133">
+              % <ref xml:id="bHY6yh-de5998-Rd1e130">Idem 23 capitulo</ref> <quote xml:id="bHY6yh-de5998-Qd1e137">primi<lb ed="#C" break="no"/>tiis 
+              frugum terre tue in domum domini dei tui</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e144">
+              % <ref xml:id="bHY6yh-de5998-Rd1e141">Idem 25</ref> <quote xml:id="bHY6yh-de5998-Qd1e146">loquere filiis
+              <lb ed="#C"/>ut collat mihi primitias ab omni homine qui offert
+              <lb ed="#C"/>ultro accipietis eas</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e154">
+              % <quote xml:id="bHY6yh-de5998-Qd1e158">In leuitico capitulo poenaliter
+              <lb ed="#C"/>Omnes decime terre defrugibus suis de po<lb ed="#C" break="no"/>mis 
+              domini sunt et illi sanctificantur</quote>
+            </p>
+            <!--manuscript makes the next paragraph a continuation-->
+            <p xml:id="bHY6yh-de5998-d1e166">
+              <quote xml:id="bHY6yh-de5998-Qd1e168">bo<lb ed="#C" break="no"/>ues 
+              oues et capre que pastoris uirga tran<lb ed="#C" break="no"/>seunt 
+              quicquid uenerit decimum sanctificabitur deo</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e176">
+              <lb ed="#C"/>% <ref xml:id="bHY6yh-de5998-Rd1e177">In libro numeri capitulo 18</ref> <quote xml:id="bHY6yh-de5998-Qd1e180">Locutus est dominus ad a<lb ed="#C" break="no"/>aron 
+              et caetera fili leui deduces decimas israelis in
+              <lb ed="#C"/>possessionem pro ministerio quo seruiunt mihi in taber<lb ed="#C" break="no"/>naculo 
+              federis</quote>
+            </p>
+            <!--manuscript makes the next paragraph a continuation-->
+            <p xml:id="bHY6yh-de5998-d1e190"> /
```

Comment:
is this slash left over from the venice transcription? If it is, please delete. If not, please just confirm that it should be there.